puppet/zulip_ops/manifests/base.pp

```diff
@@ -19,9 +19,8 @@
                         # Needed for zulip-ec2-configure-network-interfaces
                         "python3-six",
                         "python-six",
-                          # This one is needed for postgres as well
                         # "python3-boto", # missing on trusty
```

Comment:
I would rather just upgrade the version of `wal-e` to the point that defaults to Python 3 to solve the issue.  It's a smaller change to the setup.